src/app/physics/drag/lesson/DragLesson.tsx

```diff
@@ -0,0 +1,798 @@
+      <QuizQuestion
+        question="When you release a ball and it starts to fall (so it's velocity is 0, but its acceleration is downward), what are all the forces on that ball?"
+        choices={[
+          {
+            text: "There is a downward drag force causing the ball to start moving down.",
+            isCorrect: false,
+            explanation: "Not quite, the downward force is gravity.",
+          },
+          {
+            text: "There is a downward gravitational force and an upward drag force.",
+            isCorrect: false,
+            explanation:
+              "Not quite, since it's not moving yet, there wouldn't be a drag force. Drag is resistive against motion, so there should not be any drag until an object starts moving.",
+          },
+          {
+            text: "There is a downward gravitational force and no others.",
+            isCorrect: true,
+            explanation:
+              "Exactly! Gravity acts on the object, but since velocity is zero, there is no drag.",
+          },
+          {
+            text: "Velocity is 0, so there are no forces.",
+            isCorrect: false,
+            explanation:
+              "Not quite! If acceleration were 0, that would mean that there is no net force, but the ball is accelerating downward so there is force.",
+          },
+        ]}
+      />
+      <QuizQuestion
+        question="What is terminal velocity?"
+        choices={[
+          {
+            text: "Terminal velocity is the velocity at the end of the problem, often when t = 1s.",
+            isCorrect: false,
+            explanation: "Not quite.",
+          },
+          {
+            text: "Terminal velocity is the top speed.",
+            isCorrect: false,
+            explanation:
+              "Not quite, we can have an object that never reaches terminal velocity (and would thus have a lower top speed) and we can have objects that exceed their terminal velocity.",
+          },
+          {
+            text: "Terminal velocity is the velocity when the forces cancel out.",
+            isCorrect: true,
+            explanation:
+              "Exactly! Terminal velocity is when the net acceleration is 0. If an object is below terminal velocity, gravity is stronger than drag, so it speeds up. And if an object is above terminal velocity, gravity is weaker than drag, so it slows down.",
+          },
+        ]}
+      />
+    </Block>,
+
+    // Slide 6: Choice of Skipping Optional Section Slide
+    <Block color="green" title="Optional Section Choice">
+      <p>
+        There are a few slides that dive deep into the source of drag and where
+        the equations come from.{" "}
+        <Emphasize>
+          If you'd like to learn it all, click the arrow at the top right to
+          continue. Otherwise,{" "}
+          <a className="text-blue-500" href="?slide=11">
+            click here to skip the optional section
+          </a>
```

Comment:
nit: I like the anchor tag here, but the default behavior does push a new frame onto the browser's navigation stack, which is different from how the arrow keys behave